omnipod/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/OmnipodFragment.kt

```diff
@@ -279,6 +280,15 @@ class OmnipodFragment : DaggerFragment() {
             omnipod_tempbasal.text = activePlugin.activeTreatments
                 .getTempBasalFromHistory(System.currentTimeMillis())?.toStringFull() ?: "-"
 
+            // total delivered
+            omnipod_total_delivered.text = podStateManager.totalInsulinDelivered.let {
+                when {
+                    it == null                        -> "-"
+                    it < OmnipodConst.POD_SETUP_UNITS -> resourceHelper.gs(R.string.omnipod_total_delivered_pod_preparing)
```

Comment:
For consistency and simplicity I think it would be better to show '-' if !podStateManager.isPodActivationCompleted()

Reply:
That's true, I'll do that